diff --git a/giflib/Makefile b/giflib/Makefile
index 793870e..7706703 100644
--- a/giflib/Makefile
+++ b/giflib/Makefile
@@ -11,16 +11,16 @@ GIF_INC ?= /usr/include
    USR_LIBS += 114135a4-6f6c-11d3-95bc-00104b8742df
 
    USR_LIBS_Linux += Xm Xt Xp Xtst X11
-   USR_SYS_LIBS_Linux += pthread dl ungif gif
+   USR_SYS_LIBS_Linux += pthread dl gif
 
    USR_LIBS_Darwin += Xm Xt Xp Xtst X11
-   USR_SYS_LIBS_Darwin += pthread dl ungif gif
+   USR_SYS_LIBS_Darwin += pthread dl gif
 
    USR_LIBS_solaris += Xm Xt Xmu X11 Xext
-   USR_SYS_LIBS_solaris += pthread dl rt ungif gif
+   USR_SYS_LIBS_solaris += pthread dl rt gif
 
    USR_LIBS_hpux11_11_mt = Xm Xt X11 Xext
-   USR_SYS_LIBS_hpux11_11_mt = pthread ungif gif
+   USR_SYS_LIBS_hpux11_11_mt = pthread gif
 
    LIBRARY = cf322683-513e-4570-a44b-7cdd7cae0de5
 
